crates/nu-utils/src/sample_config/default_env.nu

```diff
@@ -69,17 +69,13 @@ $env.ENV_CONVERSIONS = {
 }
 
 # Directories to search for scripts when calling source or use
-#
-# By default, <nushell-config-dir>/scripts is added
 $env.NU_LIB_DIRS = [
-    ($nu.default-config-dir | path join 'scripts')
+    # ($nu.default-config-dir | path join 'scripts') # add <nushell-config-dir>/scripts
```

Comment:
I think this should be uncommented as well. or if it's strictly not the default, we should make it the default in the rust code somewhere. we need to have a nu_lib_dirs place as a default i think.

Reply:
yeah, agree :thinking:

Reply:
maybe we should keep that for another PR? it's getting quite large already :relieved:

Reply:
(same answer for the `NU_PLUGIN_DIRS` thread i closed below)

Reply:
at least now they have the same value as in `nu -n` :+1:

Reply:
We're making all these changes here. I'm not sure removing two `#` is too big for this PR. I vote that we uncomment these and leave them in place for this PR. If we want to change it later, and break from our defaults that we've had for so long, then we can decide that later.

Reply:
so we go and **change the Rust code?**
because simply uncommenting this two lists will get us back in the state before the PR where using the default config and `nu -nu` does not give the same behaviour :thinking:

Reply:
I think that's for the team to decide but in my opinion, yes, we should change the rust code to add this.